cli/src/commands/auth/status.ts

```diff
@@ -1,5 +1,4 @@
 import { BaseCommand } from '../../base.js';
-import { getProfile } from '../../credentials.js';
 
 export default class Status extends BaseCommand {
```

Comment:
Could we add a log message for `this. apiKeyLocation`?

I have changed the `.env` and the shell env variable to something not valid and it still logs me in 😢 

![image](https://user-images.githubusercontent.com/2181866/180988968-c26f7082-e3e5-4d11-9b54-58b06a84bc6c.png)

Reply:
`xata auth status` only checks the credentials file, as all subcommands of `xata auth`.

I am thinking on doing maybe a `xata status` command for project-level configuration that tells you a few things:
- If you have a project configured and the path of the configuration file.
- The resolved database URL.
- The resolved branch name.
- Checking the api key in the env file.

But, like I said, it is expected that `xata auth status` only checks the credentials file.